org.kde.kdenlive.json

```diff
@@ -0,0 +1,466 @@
+                }
+            ]
+        },
+        {
+            "name": "kfilemetadata",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DENABLE_TESTING=OFF",
+                "-DCMAKE_INSTALL_LIBDIR=lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/frameworks/5.41/kfilemetadata-5.41.0.tar.xz",
+                    "sha256": "714b78e075023a8047595ae8821d073e373e1bc0c283c044741a29bc50503e79"
+                }
+            ]
+        },
+        {
+            "name": "eigen",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release"
+            ],
+            "cleanup": [
+                "/share"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://bitbucket.org/eigen/eigen/get/3.3.4.tar.bz2",
+                    "sha256": "dd254beb0bafc695d0f62ae1a222ff85b52dbaa3a16f76e781dce22d0d20a4a6"
+                }
+            ]
+        },
+        {
+            "name": "fftw",
+            "config-opts": [
+                "--disable-static",
+                "--enable-shared",
+                "--disable-doc"
+            ],
+            "cleanup": [
+                "/bin"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://www.fftw.org/fftw-3.3.7.tar.gz",
+                    "sha256": "3b609b7feba5230e8f6dd8d245ddbefac324c5a6ae4186947670d9ac2cd25573"
+                }
+            ]
+        },
+        {
+            "name": "movit",
+            "config-opts": [
+                "--disable-static",
+                "GTEST_DIR=./googletest"
+            ],
+            "make-args": [
+                "GTEST_DIR=./googletest"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://movit.sesse.net/movit-1.5.3.tar.gz",
+                    "sha256": "ead191d717dbefbe914260ed0335cf04a7dfc6fde8f23293c5ca7f88e45ace81"
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/google/googletest/archive/release-1.8.0.tar.gz",
+                    "sha256": "58a6f4277ca2bc8565222b3bbd58a177609e9c488e8a72649359ba51450db7d8",
+                    "dest-filename": "gtest.tar.gz"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "tar xf gtest.tar.gz --strip 1"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "libsamplerate",
```

Comment:
In order to fix this on aarch64 you'll need to rm-configure: true and autoreconf e.g. https://github.com/flathub/shared-modules/blob/9cc549960171590d6a0736d762c083a51d1c5779/udev/udev-175.json